osbs/core.py

```diff
@@ -59,15 +58,12 @@ def __init__(self, openshift_api_url, openshift_oauth_url, kubelet_base, verbose
     def os_oauth_url(self):
         return self._os_oauth_url
 
-    def _build_url(self, url, namespace=None):
-        if namespace:
-            url += "?namespace=%s" % namespace
-            return urlparse.urljoin(self.os_api_url, url)
-        else:
-            return urlparse.urljoin(self.os_api_url, url)
-
-    def _build_k8s_url(self, url):
-        return urlparse.urljoin(self.kubelet_base, url)
+    def _build_url(self, url, **query):
+        query = [k + '=' + str(v) for k, v in query.items()]
```

Comment:
Am checking if urlparse doesn't have method for this but looks like it doesn't =/